bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/PowerControlPower.java

```diff
@@ -69,13 +68,42 @@ public void onReceiveCommand(String channelId, LGWebOSHandler handler, Command c
                         }, i * WOL_PACKET_RETRY_DELAY_MILLIS, TimeUnit.MILLISECONDS);
                     }
                 }
+            } else {
+                switch (state) {
+                    case CONNECTING:
+                    case REGISTERING:
+                    case REGISTERED:
+                        logger.debug("Received ON - TV is already on.");
+                        handler.postUpdate(channelId, OnOffType.ON);
+                        break;
+                    case DISCONNECTING:
+                        logger.debug("Received ON - TV is currently disconnecting.");
```

Comment:
As already mentioned, I disagree here. We nned the ability to turn on the TV with WOL when in this state.

Reply:
but it does not work. the tv does not react to WOL when it is on and specifically when it is in the shutdown process. WOL will not stop a running shutdown process